CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Change Log
 ==========
+Version 1.7.0
```

Comment:
`Version 1.7.0-SNAPSHOT` ;) + we should make appropriate changes in `gradle.properties`